x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rules.tsx

```diff
@@ -34,20 +29,19 @@ export interface UseRules {
 export const useRules = ({
   pagination,
   filterOptions,
-  refetchPrePackagedRulesStatus,
   dispatchRulesInReducer,
 }: UseRules): ReturnRules => {
   const [rules, setRules] = useState<FetchRulesResponse | null>(null);
   const reFetchRules = useRef<(refreshPrePackagedRule?: boolean) => void>(noop);
   const [loading, setLoading] = useState(true);
   const [, dispatchToaster] = useStateToaster();
 
-  const filterTags = filterOptions.tags?.sort().join();
```

Comment:
I think there's one more `filterOptions.tags?` in `x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/api.ts` that can be removed.

Reply:
Looks like it. I updated that one. Thanks for finding it